code/modules/mining/ores_coins.dm

```diff
@@ -17,6 +17,33 @@
 			user << "<span class='info'>Not enough fuel to smelt [src].</span>"
 	..()
 
+/obj/item/weapon/ore/Crossed(atom/movable/AM)
+	var/obj/item/weapon/storage/bag/ore/OB
+	var/turf/open/floor/F = get_turf(src)
+	if(loc != F)
+		return ..()
+	if(ishuman(AM))
+		var/mob/living/carbon/human/H = AM
+		for(var/thing in H.get_storage_slots())
+			if(istype(thing, /obj/item/weapon/storage/bag/ore))
+				OB = thing
+				break
+	else if(ishuman(AM))

```

Comment:
you're checking ishuman() twice, and since you're using "else if" it will never be run since the ishuman() above will always be called instead.